src/composables/useBcrosAuth.ts

```diff
@@ -44,9 +44,12 @@ export const useBcrosAuth = () => {
           // successfully initialized so setup other pieces
           keycloak.syncSessionStorage()
           keycloak.scheduleRefreshToken()
+
           // set user info
-          console.info('Setting user name...')
-          await account.setUserName()
+          // NB. user name will be set automatically; calculated from the kcUser initialized in initKeyCloak function
+          // console.info('Setting user name...')
+          // await account.setUserName()
```

Comment:
Please delete commented-out code some time.

Comments explaining why something is the way it is are good 👍

Reply:
Good point. I will delete these lines in my next PR.